src/components/DropdownMenu.tsx

```diff
@@ -1,21 +1,23 @@
 import React, { FC } from 'react';
+
 import { DropdownMenuProps } from './types';
 
 const DropdownMenu: FC<DropdownMenuProps> = (props) => {
-    const { dropdown, registerFunction, registerName } = props;
 
+    const { dropdown, registerFunction, registerName } = props;
+    
     return (
-        <div className="m-2 ml-4 flex-initial pl-2 ">
-            <select {...registerFunction(registerName)} className="mt-1 h-10 w-56 rounded-md
-                         border border-gray-300 bg-white p-2 text-sm leading-5 text-gray-400
-                         hover:border-blue-500">
+        <div className="m-2 flex-initial pl-2 xl:flex-initial ">
+            <select {...registerFunction(registerName)} className="mt-1 h-10 w-56 rounded-md border
+             border-gray-300 bg-white p-2 text-sm leading-5 text-gray-400 focus:outline-none" defaultValue=''>
+                <option value='' disabled hidden >Pick a choice!</option>
                 {dropdown.map(element => {
                     return (
-                        <option key={element} value={element}>{element}</option>
+                        <option key={element.id} value={element.id}>{element.name}</option>  
```

Comment:
destructure the individual elements prior on top as const { id, name } = element;

use 'id' and 'name' in place of 'element.id' and 'element.name'